news/_posts/2023-10-10-release-20231010.md

```diff
@@ -0,0 +1,44 @@
+---
+kind: article
+title: "Release 20231010 and the Community"
+author: "The OpenRA Team"
+created_at: 2023-10-10 18:00 +0000
+disqus_id: "release-20231010"
+permalink: "/news/release-20231010/"
+---
+
+With two months of successful playtesting behind us, we are pleased to finally announce [Release 20231010]({{ '/download' | relative_url }}), which brings the public release up to date with development changes from November 2022 through to August 2023. The new release contains some long-awaited UI and UX improvements and many polish changes overall. It also serves as a jump-off point for many of the larger upcoming features, so do expect great things in the future!
+
+<figure>
+   <video autoplay loop muted>
+    <source src="{{ '/images/news/20231010-darkcolors.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20231010-darkcolors.webm' | relative_url }}" type="video/webm">
+    <img src="{{ '/images/news/20231010-darkcolors.png' | relative_url }}"/>
+  </video>
+  <figcaption>Restored the ability to select darker player colors.</figcaption>
+</figure>
+
+For more information see the other newsposts [[one]({{ '/news/playtest-20230801/' | relative_url }}), [two]({{ '/news/playtest-20230808/' | relative_url }})], [three]({{ '/news/playtest-20230927/' | relative_url }})] and, as always, for a full overview - [the full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/08829bc0e6a1bf62aa7797fcfc0a9810d27e81d2).
+
+While OpenRA's releases have slowed down due to lack of manpower, our third party scene is going strong! An OpenRA mod & standalone game [Combined Arms](https://www.moddb.com/mods/command-conquer-combined-arms) was the second most popular on ModDB in June this year. It includes units from various C&C games and has 5 factions, each having 5 subfactions. Its newest installment contains 25 singleplayer missions. [DORF Real-Time Strategic Conflict](https://www.dorf-rts.com), a game based on OpenRA's engine has become one of the most anticipated RTS's. We highly encourage everyone to checkout its gameplay trailer as well as [other videos](https://www.youtube.com/@dorf-rts).
```

Comment:
```suggestion
While OpenRA's releases have slowed down due to lack of manpower, our third party scene is going strong! An OpenRA mod & standalone game [Combined Arms](https://www.moddb.com/mods/command-conquer-combined-arms) was the second most popular on ModDB in June this year. It includes units from various C&C games and has 5 factions, each having 5 subfactions. Its newest installment contains 25 singleplayer missions. [DORF Real-Time Strategic Conflict](https://www.dorf-rts.com), a game based on OpenRA's engine has become one of the most anticipated RTSes. We highly encourage everyone to checkout its gameplay trailer as well as [other videos](https://www.youtube.com/@dorf-rts).
```

It's a plural not a possessive form.

Reply:
when not in possession it seems the correct way of writing shortforms it seems is just an s without an apostrophe

Reply:
Except in cases where the word already ends with an `s`.

Reply:
https://www.merriam-webster.com/grammar/how-to-use-plural-s-and-es-exceptions-grammar

Reply:
We could also avoid that by writing `one of the most anticipated RTS releases` (or `games` instead of `releases`).

Reply:
I'm going with the latter suggestion, as RTSes looks weird. I don't think that rule applies to shortforms too